test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo_test_helper
```

Comment:
Why the `DON'T MERGE` in the commit message ?

Reply:
this module requires `odoo_test_helper` lib to run [testcase](https://github.com/OCA/partner-contact/blob/e1cff143a9e7fd7927e8a1394731f9a21aa3051e/partner_identification/tests/test_res_partner.py#L4) but odoo requirements.txt does not have `odoo_test_helper` lib

Reply:
Yes the change is correct, and we want it to be merged.
But the title of the commit needs to be changed by removing `[DON'T MERGE]`
This convention is used when we need to add a temporary reference to an existing PR more information in https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference(s)-to-another-pull-request(s)